instrumentation/dubbo/dubbo-apache-2.7/javaagent/dubbo-apache-2.7-javaagent.gradle

```diff
@@ -0,0 +1,17 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group = "org.apache.dubbo"
+    module = "dubbo"
+    versions = "[2.7.0,)"
+  }
+}
+
+dependencies {
+  implementation project(':instrumentation:dubbo:dubbo-apache-2.7:library')
+
+  library group: 'org.apache.dubbo', name: 'dubbo', version: '2.7.0'
```

Comment:
We target tests to oldest version supported

Reply:
Before Dubbo 2.7.0, it is called alibaba Dubbo.And it is called Apache Dubbo since 2.7.0.